package.json

```diff
@@ -277,7 +277,6 @@
 		"stackframe": "^1.1.1",
 		"stacktrace-gps": "^3.0.3",
 		"stylelint": "^14.11.0",
-		"stylelint-config-standard-scss": "^5.0.0",
```

Comment:
This means that we can also remove `postcss-scss` from the dependencies.

Reply:
Done